.bumpversion.cfg

```diff
@@ -0,0 +1,8 @@
+[bumpversion]
```

Comment:
What is the purpose of this file ?

Reply:
Sorry, I didn't see you were online when I was working in this pull request :o). The purpose of this file is to automate the new version release process with the usage of the package [bumpversion](https://pypi.org/project/bumpversion/). These are suggestions I thought to advance the package in some other dimensions. We can work on a clean pull request.

Reply:
RIght, my main concern about this PR is that this project doesn't require new version releasing processes, such as this one. I'm afraid it would just confuse some more theoretical users. Beside, I currently don't have much time to update the project.

Reply:
> 
> 
> RIght, my main concern about this PR is that this project doesn't require new version releasing processes, such as this one. I'm afraid it would just confuse some more theoretical users. Beside, I currently don't have much time to update the project.

I think we can achieve a great release cycle completely automated. Files like this are handy in this regard. Would making this file hidden suffice?

Reply:
Like I said, the real issue here is that I don't see any reason to set up an automated release cycle. This repo is not made to be a super functional tool with updates and upgrades twice a month, it's merely the results of my work and research on the Transformer.

If you think there is an opportunity to keep maintaining and upgrading this package in the future, I encourage you to keep improving and building on your fork, the results are quite impressive ! But I'm afraid too much CI/CD tools in this repo would mislead on its objectives.

Reply:
Whichever you think is best for the project, I'm with you.